test/unit/handlers/authorizations.test.js

```diff
@@ -18,7 +18,7 @@ const Hapi = require('@hapi/hapi')
 const ErrorHandler = require('@mojaloop/central-services-error-handling')
 const Logger = require('@mojaloop/central-services-logger')
 
-const Mockgen = require('../../util/mockgen').mockRequest
+const Mockgen = require('../../util/mockgen.js')
```

Comment:
```suggestion
const Mockgen = require('../../util/mockgen')
```

there is no need for `.js` extension